docs/manual/src/proc_macro/index.md

```diff
@@ -104,6 +104,12 @@ will fail).
 pub struct MyRecord {
     pub field_a: String,
     pub field_b: Option<Arc<MyObject>>,
+    // Fields can have a default value.
+    // Currently, only string, integer, float and boolean literals are supported as defaults.
```

Comment:
I wonder if it is worth adding something like "these work the same way as documented for UDL files" or similar (the docs need a restructure, but udl/structs.md does describe this in some detail)